ShellScript/Bash.sublime-syntax

```diff
@@ -27,22 +28,22 @@ file_extensions:
   - .zshrc
 
 first_line_match: |
-  (?x)                     # ignore whitespace in this regex
-    ^                      # assert start of the line
-    \#!                    # shebang token
-    .*                     # any characters
-    \b(bash|zsh|sh|tcsh)\b # "bash" or "zsh" or "sh" or "tcsh"
-    |                      # ... or ...
-    ^                      # assert start of the line
-    \#                     # literal "#" character
-    \s*                    # any whitespace
-    -\*-                   # literal "-*-" string
-    [^*]*                  # any characters except "*"
-    mode:                  # literal "mode:" string
-    \s*                    # any whitespace
-    shell-script           # literal "shell-script" string
-    [^*]*                  # any characters except "*"
-    -\*-                   # literal "-*-" string
+  (?x)                         # ignore whitespace in this regex
+    ^                          # assert start of the line
+    \#!                        # shebang token
+    .*                         # any characters
+    \b(bash|zsh|sh|tcsh|ash)\b # "bash" or "zsh" or "sh", "tcsh" or "ash"
```

Comment:
I think you could have just changed `bash` into `b?ash`

Reply:
I prefer the explicit version here. It's more readable that way.